OpenEdXMobile/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -190,6 +191,14 @@ public void onClick(View view) {
             }
         });
 
+        drawerNavigationBinding.drawerHelpCenter.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                ((BaseFragmentActivity) getActivity()).closeDrawer();
```

Comment:
- A condition should be placed here just to check if HelpCenter activity is not already opened only then we need to start this activity otherwise it will open multiple activities each time user clicks HelpCenter option.
- StartActivity part for HelpCenter should be moved to Router as per standards.

Reply:
Help Center Activity is always opened on top of current activity and does not contains nav drawer, So I think a check would be useless here.

Reply:
👍